DashAI/front/src/components/ResponsiveAppBar.jsx

```diff
@@ -0,0 +1,127 @@
+import * as React from "react";
+import AppBar from "@mui/material/AppBar";
+import Box from "@mui/material/Box";
+import Toolbar from "@mui/material/Toolbar";
+import IconButton from "@mui/material/IconButton";
+import Typography from "@mui/material/Typography";
+import Menu from "@mui/material/Menu";
+import MenuIcon from "@mui/icons-material/Menu";
+import Container from "@mui/material/Container";
+import Avatar from "@mui/material/Avatar";
+import Button from "@mui/material/Button";
+import MenuItem from "@mui/material/MenuItem";
+import { Link as RouterLink, useLocation } from "react-router-dom";
+import { useTheme } from "@mui/material/styles";
+import HomeIcon from "@mui/icons-material/HomeOutlined";
+
+const pages = [
+  { name: "Data", to: "/data", disabled: false },
+  { name: "Experiments", to: "/experiments", disabled: false },
+  { name: "Results", to: "/results", disabled: false },
+  { name: "Play", to: "/play", disabled: true },
```

Comment:
For the serving from the backend works, path starts with /app/, i.e
```suggestion
  { name: "Data", to: "/app/data", disabled: false },
  { name: "Experiments", to: "/app/experiments", disabled: false },
  { name: "Results", to: "/app/results", disabled: false },
  { name: "Play", to: "/app/play", disabled: true },
```

Reply:
I forgot about it. Changed!